testing/tcolors/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Joe Searle <joe@jsearle.net>
+# Maintainer: Joe Searle <joe@jsearle.net>
+pkgname=tcolors
+pkgver=0.3.1
+pkgrel=0
+pkgdesc="Commandline color picker and palette builder"
+url="https://github.com/bcicen/tcolors"
+arch="all"
+license="MIT"
+makedepends="go"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/bcicen/tcolors/archive/v$pkgver.tar.gz"
+options="!check" # No tests
+
+build() {
+	go mod download
+	go build -ldflags "-w -X main.version=${pkgver}-${pkgrel} -X main.build=$pkgbuild" -o tcolors
+}
+
+package() {
+	install -Dm755 tcolors "${pkgdir}/usr/bin/tcolors"
+	install -Dm644 docs/man1/tcolors.1 "$pkgdir/usr/share/man/man1/${pkgname}.1"
+	install -Dm644 LICENSE -t "$pkgdir/usr/share/licenses/$pkgname"
```

Comment:
Ah, no need to install licenses for SPDX licenses (so all licenses listed on https://spdx.org/licenses)

Reply:
corrected :+1: